Source/UINavigation/Public/UINavPCComponent.h

```diff
@@ -133,7 +133,7 @@ class UINAVIGATION_API UUINavPCComponent : public UActorComponent
 
 	void CallCustomInput(const FName ActionName, const bool bPressed);
 
-	void OnControllerConnectionChanged(bool bConnected, FPlatformUserId UserId, int32 UserIndex);
+	virtual void OnControllerConnectionChanged(EInputDeviceConnectionState NewConnectionState, FPlatformUserId PlatformUserId, FInputDeviceId InputDeviceId);
```

Comment:
Just noticed, this should probably have an override at the end

Reply:
No I made it as a virtual to allow the possibility to override it in a child class of UINavPCComponent.
But if you want me, I can simply remove the Virutal on it!

What do I do?

Reply:
But isn't this a function from UINavPCReceiver?

Reply:
No, it's declared in line 86, with the "PlatformInputMapper.GetOnInputDeviceConnectionChange()"
It's not a native function of Unreal. 

Then in line 317, you execute the portion of code of the Interface "IUINavPCReceiver".

I've removed the virtual, no big deal about it